source/docs/articles/sites/migrate/wordpress-site-networks.md

```diff
@@ -0,0 +1,136 @@
+ENV='ENV'
+SITE='SITEID'
+
+read -sp "Your Pantheon Password: " PASSWORD
+if [[ -z "$PASSWORD" ]]; then
+echo "Woops, need password"
+exit
+fi
+
+while [ 1 ]
+do
+sshpass -p "$PASSWORD" rsync --partial -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222'  $ENV.$SITE@appserver.$ENV.$SITE.drush.in:files/* --temp-dir=../tmp/  ./files/
+if [ "$?" = "0" ] ; then
+echo "rsync completed normally"
+exit
+else
+echo "Rsync failure. Backing off and retrying..."
+sleep 180
+fi
+done
+```
+This script connects to your Pantheon site's Dev environment and starts uploading your files. If an error occurs during transfer, rather than stopping, it waits 180 seconds and picks up where it left off.  
+
+If you are unfamiliar or uncomfortable with bash and rsync, an FTP client that supports SFTP, such as FileZilla, is a good option. Find your Dev environment's SFTP connection info and connect with your SFTP client. Navigate to `~/code/wp-content/uploads/`. You can now start your file upload.  
+
+## Database  
+
+**Database** - a single `.sql` dump that contains the content and active state of the site's configurations.
+
+You'll need a .sql file containing the data from the site you want to import. If you haven't done so already, make sure you remove any data from the cache tables. That will make your .sql file much smaller and your import that much quicker.
+
+If your `.sql` file is less than 500MB, you can use the Import tool on the Workflow tab to import the database from a URL. If it is less than 100MB, you can upload the file directly. Larger files require the use of the command line to import the data.
+
+1. From the Dev environment on the Site Dashboard, click **Connection Info** and copy the Database connection string. It will look similar to this:
+
+ ```
+ mysql -u pantheon -p{massive-random-pw} -h dbserver.dev.{site-id}.drush.in -P {site-port} pantheon
+ ```
+2. From your terminal, `cd` into the directory containing your `.sql` archive. Paste the connection string and append it with:
+`< database.sql`
+Your command will now look like:
+
+ ```
+ mysql -u pantheon -p{massive-random-pw} -h dbserver.dev.{site-id}.drush.in -P {site-port} pantheon < database.sql
+ ```
+3. After you run the command, the .sql file is imported into your Pantheon Dev database.  
+
+You should now have all three of the major components of your site imported into Pantheon.
+
+## WP Search and Replace
+
+When you imported your database, all of the URL's remained active at the previous site's domain name. Visiting the site at this point should return an incorrect connection information error message. To resolve it, the last step of the import process is to change the URL's to match the development environment using the wp-cli command, `wp search-replace`. In the example below, replace `example.com` with the Domain your site currently runs at.
+
+```bash
+terminus wp search-replace example.com dev-example-network.pantheon.io --url=example.com --network --site=example-network --env=dev

```

Comment:
Worth mentioning you can use `--dry-run` to preview results, or `--verbose` to get more verbosity.